doc/rbd/rbd-windows.rst

```diff
@@ -142,6 +142,34 @@ initializes a partition::
     # Show the partition letter (for example, "D:" or "F:"):
     (Get-Partition -DiskNumber $diskNumber).DriveLetter
 
+SAN policy
+----------
+
+The Windows SAN policy determines which disks will be automatically mounted.
+The default policy (``offlineShared``) specifies that:
+
+  All newly discovered disks that do not reside on a shared bus (such as SCSI
+  and iSCSI) are brought online and made read-write. Disks that are left
+  offline will be read-only by default."
+
+Note that recent WNBD driver versions report rbd-wnbd disks as "SAS", which is
+considered a shared bus. As a result, the disks will be offline and read-only
+by default.
+
+In order to turn a disk online and clear the read-only flag, use the following
+commands::
+
+  set-disk -Number $diskNumber -IsOffline $false
+  set-disk -Number $diskNumber -IsReadOnly $false
```

Comment:
I know PowerShell is mostly case insensitive, but it's capitalized elsewhere in the doc:
```suggestion
  Set-Disk -Number $diskNumber -IsOffline $false
  Set-Disk -Number $diskNumber -IsReadOnly $false
```